### PR TITLE
Print warning messages for stale open handle instead of removing it

### PR DESCRIPTION
**Pull request by huanghua78**
### What changes are proposed in this pull request?

Checking all open handles periodically. 
if some handle is stale (inactive for more than 24 hours), please warning messages.
Every time an open handle is used, its last access time is updated.

In future, we may need to handle this case instead of printing warning messages.

### Why are the changes needed?

If an open handle is inactive for a long time, it may be caused by stuck, or by disconnected client.
So a warning message is generated. Originally the open handle was closed in such case.
But if a write is really slow or paused there, we are not going to close this handle.

### Does this PR introduce any user facing changes?

N/A.

### CODE REVIEW

File: dora/core/server/worker/src/main/java/alluxio/worker/dora/DoraOpenFileHandleContainer.java [hunk 1]
```diff
@@ -40,14 +40,19 @@ public class DoraOpenFileHandleContainer extends Thread {
...
+        LOG.debug("Checking all handles ...");
         // Iterate the mOpenFileHandles if some handles are stale (not active for a long time).
         for (Map.Entry<String, OpenFileHandle> entry : mOpenFileHandles.entrySet()) {
           String key = entry.getKey();
           OpenFileHandle handle = entry.getValue();
-          if (System.currentTimeMillis() - handle.getLastAccessTimeMs() >= Constants.HOUR) {
-            mOpenFileHandles.remove(key);
-            handle.close();
+          LOG.debug("Checking handle for {}", handle.getPath());
+          if (System.currentTimeMillis() - handle.getLastAccessTimeMs() >= Constants.HOUR * 24) {
```
Comment (elega) [inline dora/core/server/worker/src/main/java/alluxio/worker/dora/DoraOpenFileHandleContainer.java:54]: i'm thinking if we should make this configurable

Comment (huanghua78) [reply dora/core/server/worker/src/main/java/alluxio/worker/dora/DoraOpenFileHandleContainer.java:54]: With this patch, we are not going to close the handle. It only reports warning message.
I don't want to add configuration keys at this moment. We have had too many keys.

When we really to cleanup the stale open handles, we may add configurable keys. That is a future task.
To enable clean up stable open handles, we need connection/session support.
When a session is ended/terminated, all the open handles in this session should be cleaned up.

**Review by elega — APPROVED** [review]

**Review by JiamingMai — APPROVED** [review]
LGTM

**Comment by huanghua78** [comment]
alluxio-bot, merge this please.

**Comment by jiacheliu3** [comment]
alluxio-bot, cherry-pick this to branch-3-1.1.x please

**Pull request merged**